analyticsclient/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '0.15.3'
+__version__ = '0.15.4'
```

Comment:
@feanil  As it was already updated to python3 do we still need to release a new version.